client/src/partials/templates/modals/dateInterval.tmpl.html

```diff
@@ -0,0 +1,60 @@
+<form name="DateInterval" novalidate>
+  <div class="modal-header">
+    <h4>
+      <i class="fa fa-search"></i> &nbsp;
+      {{ 'FORM.SELECTS.SELECT_DATE_INTERVAL' | translate }}
+      <span ng-if="$ctrl.selected" class="label label-warning">{{ $ctrl.selected | translate }}</span>
+    </h4>
+  </div>
+
+  <div class="modal-body">
+    <label for=""></label>
+    <div class="form-group text-center">
+      <i class="fa fa-calendar"></i>
+      <div class="btn-group">
+        <button class="btn btn-default" type="button"
+          ng-repeat="option in $ctrl.options"
+          ng-click="$ctrl.search(option)">
+          {{ option.label | translate }}
+        </button>
+      </div>
+    </div>
+
+    <div class="text-left">
+      <div class="form-group">
+        <bh-date-editor
```

Comment:
Could you take a look at the patient search and invoice search modals for inspiration for dates?  They handle begin and end dates very clearly.  I've attached a screenshot below.

![dateinput](https://cloud.githubusercontent.com/assets/896472/16620970/74d5a346-438c-11e6-9921-609b28046b44.png)